examples/rllib/self_play_train.py

```diff
@@ -103,12 +104,16 @@ def env_creator(env_config):
   config["model"]["lstm_use_prev_reward"] = False
   config["model"]["lstm_cell_size"] = 256
 
-  # 6. Initialize ray and trainer object
-  ray.init(num_cpus=num_cpus + 1)
```

Comment:
What's the logic to remove the explicit number of CPUs? Is this better?

Reply:
When using tune I did not have enough processes for the job to run, so it was stuck in pending. The default (number of virtual cores) worked. So we can increase the num_cpus from 1, or we can leave it on the default value.